seqr/views/utils/project_context_utils.py

```diff
@@ -110,11 +110,12 @@ def add_child_ids(response):
         family['individualGuids'] = individual_guids_by_family[family['familyGuid']]
 
 
-def families_discovery_tags(families):
+def families_discovery_tags(families, project=None):
     families_by_guid = {f['familyGuid']: dict(discoveryTags=[], **f) for f in families}
 
+    family_filter = {'family__project': project} if project else {'family__guid__in': families_by_guid.keys()}
```

Comment:
removes a join with the project table when unneded (querying directly on the project uses `family.project_id` so the project table is never joined)